src/Model/TransportModel/tsp1cnc1.f90

```diff
@@ -83,21 +91,29 @@ subroutine cnc_create(packobj, id, ibcnum, inunit, iout, namemodel, pakname)
     packobj%ncolbnd = 1
     packobj%iscloc = 1
     !
-    ! -- return
+    ! -- Give package access to the assigned labels based on dependent variable
+    packobj%tsplab => tsplab
+    !
+    ! -- Give access to governing equation scale factor
+    cncobj%eqnsclfac => eqnsclfac
```

Comment:
It does not seem like this variable is used anywhere in Cnc.  Is that correct?

Reply:
Great catch.  Probably a vestige from earlier development efforts.  Removed in d8796b4